openbb_platform/providers/polygon/openbb_polygon/models/currency_pairs.py

```diff
@@ -22,75 +25,39 @@ class PolygonCurrencyPairsQueryParams(CurrencyPairsQueryParams):
     Source: https://polygon.io/docs/forex/get_v3_reference_tickers
     """
 
-    symbol: Optional[str] = Field(
```

Comment:
Why removing this Polygon query parameters?

Reply:
Because we can do everything post-request with no real impact on performance while making it operate like everything else and simplifying it dramatically for the end-user.